contracts/interfaces/IExpressBorrow.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.18;
+
+import "../libraries/LoanLibrary.sol";
+
+interface IExpressBorrow {
+    function executeOperation(
+        address loanOriginationCaller,
+        address lender,
+        LoanLibrary.LoanTerms calldata loanTerms,
+        uint256 borrowerNet,
```

Comment:
Maybe it would be good to split this back into `principal` and `fee`?

Reply:
Yes, I like this. Originally when I tried this, I got stack to deep errors, but now that we have split out the permit functionality we have the space on the stack to do this. So added it back in! 

Also, the principal is in the loanTerms already so we dont need to re-surface the principal since it is already in the terms which we are passing through the callback.